src/App.tsx

```diff
@@ -1,57 +1,54 @@
-import {
-  AppBar,
-  Alert,
-  Box,
-  Button,
-  Container,
-  FormControl,
-  InputLabel,
-  MenuItem,
-  Select,
-  Toolbar,
-  Typography,
-  SelectChangeEvent,
-} from "@mui/material";
-import SendIcon from "@mui/icons-material/Send";
-import React from "react";
+import ActionButton from "./components/ActionButton";
+import DropDown from "./components/DropDown";
+import InstructionTable from "./components/InstructionTable";
+import React, { useState } from "react";
 import CodeMirror from "@uiw/react-codemirror";
 import { oneDark } from "@codemirror/theme-one-dark";
+import { eclipse } from "@uiw/codemirror-theme-eclipse";
 import init, { program } from "miden-wasm";
+import MidenLogo from "./MidenLogo";
+
 
 async function getExample(example: string) {
     const inputs = fetch(`https://raw.githubusercontent.com/0xPolygonMiden/examples/main/examples/${example}.inputs`)
     const masm = fetch(`https://raw.githubusercontent.com/0xPolygonMiden/examples/main/examples/${example}.masm`)
     return [(await inputs).text(), (await masm).text()];
   }
 
-function App() {
+function App(this: any) {
+
+  /**
+  * This sets the inputs to the default values.
+  */  
 
-  const [inputs, setInputs] = React.useState(
-    `{
-      "stack_init": ["0"],
-      "advice_tape": ["0"]
+  const exampleInput =     `{
+    "stack_init": ["0"],
+    "advice_tape": ["0"]
 }`
-  );
+  const [inputs, setInputs] = React.useState(exampleInput);
 
-  var exampleCode =`begin
+  const exampleCode =   `begin
   push.1
   push.2
   add
 end`
 
   const [code, setCode] = React.useState(exampleCode);
 
-  const emptyOutput = {
-    init: false,
-    success: true,
-    text: " ",
-  };
+  /**
+  * This sets the output and number of outputs to the default values. 
+  * Set to 16 now, that should change dynamically.
+  */
+
+  const emptyOutput = "\n \n \n \n";
   const [output, setOutput] = React.useState(emptyOutput);
 
-  const [numOfOutputs, setNumOfOutputs] = React.useState(1);
+  const [numOfOutputs, setNumOfOutputs] = React.useState(16);
 
-  // this should be dyncamically read from ./examples/ - buy my TS knowledge is limited
-  const examples = [
+  /**
+  * This lists all the examples available in the examples folder.
+  */  
+ const examples = [
```

Comment:
Do we still need this defined here? It's also defined in the `Dropdown` component.

Reply:
nope its a left over